test/utils/combineStates.test.ts

```diff
@@ -1,22 +1,27 @@
 import { combineStates } from '../../src/utils/combineStates';
 import { StatesType, QuerierState } from '../../src/types';
+import { QueryStateType } from '../../src';
+
+const createStateTypes = (stateTypes: Array<QuerierState>): StatesType => {
```

Comment:
@aniaw It's `createStateProps`. I have a task on my agenda to rename Querier's `States` to statuses, as `State` is super easy to confuse with React's `state`